storage/src/cache/cachedfile.rs

```diff
@@ -36,11 +37,69 @@ use crate::meta::{BlobMetaChunk, BlobMetaInfo};
 use crate::utils::{alloc_buf, copyv, readv, MemSliceCursor};
 use crate::{StorageError, StorageResult, RAFS_DEFAULT_CHUNK_SIZE};
 
+const DOWNLOAD_META_RETRY_COUNT: u32 = 5;
+const DOWNLOAD_META_RETRY_DELAY: u64 = 500;
```

Comment:
totally waiting for 2.5 seconds in a terrible network environment could not be enough. Can we make it longer like 5 seconds or 10 seconds?

Reply:
changed to 20s.

Reply:
Looks it was not changed, forget to stage changes?

Reply:
Sorry, changed to 10s instead of 20s:)